bindings/nodejs/examples/secret_manager/generate-addresses.ts

```diff
@@ -48,7 +48,7 @@ async function run() {
                 start: 0,
                 end: 4,
             },
-            internal: true,
+            options: { internal: true },
```

Comment:
Not needed in the old bindings?

Reply:
The old binding is not needed :trollface: 
Will update